README.md

```diff
@@ -1,4 +1,8 @@
-<h1 align="center"><strong>👨‍🔧👨🏻‍🔧👸🏼🍄🐲 Retro Quest 🐲🍄👸🏼👨🏻‍🔧👨‍🔧</strong></h1>
+<h1 align="center">
+<strong>
+👨‍🔧👨🏻‍🔧👸🏼🍄🐲 Retro Quest 🐲🍄👸🏼👨🏻‍🔧👨‍🔧
+</strong>
```

Comment:
Issue (Blocking): The indentation looks a bit off here. Have you installed the Prettier extension mentioned in this post? That should fix the formatting automatically. 

https://code-institute-room.slack.com/archives/C05N397K7PV/p1692220854388809
https://www.alphr.com/use-prettier-vs-code/